src/lib/utility/language-names.js

```diff
@@ -0,0 +1,19 @@
+import { LanguageModelFactory, Constants } from 'alpheios-data-models'
```

Comment:
ideally the need for this class will go away when alpheios-project/data-models#65 is complete

Reply:
I will keep an eye on it!